### PR TITLE
fixed issue #22736 - Cursor position not in right side of search keyword in mobile

### PR DESCRIPTION
**Pull request by ghost**
### Description (*)
For mobile view when search is focused I have set with only found solution https://www.google.com/search?q=jquery+set+cursor+at+end+of+input

### Fixed Issues (if relevant)
1. magento/magento2#22736: 
Cursor position not in right side of search keyword in search box when click on search again (Mobile issue)

### Manual testing scenarios (*)
1. Load website in mobile view
2. Search anything.
3. Click on search icon to focus search box
4. Check the cursor position

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/Magento/Search/view/frontend/web/js/form-mini.js [hunk 1]
```diff
@@ -133,6 +133,9 @@ define([
 
             if (this.isExpandable) {
                 this.element.attr('aria-expanded', isActive);
+                let searchValue = this.element.val();
+                this.element.val("");
+                this.element.val(searchValue);
```
Comment (VladimirZaets) [inline app/code/Magento/Search/view/frontend/web/js/form-mini.js]: This can get even simpler: this.element.focus().val(this.element.val())

**Review by VladimirZaets — CHANGES_REQUESTED** [review]
Hi @sanjaychouhan-webkul, this can get even simpler: this.element.focus().val(this.element.val())

**Review by VladimirZaets — CHANGES_REQUESTED** [review]
Hi @sanjaychouhan-webkul, this can get even simpler: this.element.focus().val(this.element.val())

**Comment by ghost** [comment]
@VladimirZaets this does not work. also getting js error because we are triggering focus() event inside onFocus event handler.
Screenshot from 2019-05-09 00-52-06we

**Comment by ghost** [comment]
@VladimirZaets also `this.element.val(this.element.val());` doesn't work.

**Review by VladimirZaets — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @VladimirZaets, thank you for the review.
[ENGCOM-5126](https://jira.corp.magento.com/browse/ENGCOM-5126) has been created to process this Pull Request

File: app/code/Magento/Search/view/frontend/web/js/form-mini.js [hunk 2]
```diff
@@ -133,6 +133,9 @@ define([
 
             if (this.isExpandable) {
                 this.element.attr('aria-expanded', isActive);
+                let searchValue = this.element.val();
```
Comment (VladimirZaets) [inline app/code/Magento/Search/view/frontend/web/js/form-mini.js]: Hi @sanjaychouhan-webkul. 
Currently, we support only ES5 syntax, so can you please change `let` to `var` and according to our [technical guidelines](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html) variables declaration should be on the top of the function.

**Review by VladimirZaets — CHANGES_REQUESTED** [review]

File: app/code/Magento/Search/view/frontend/web/js/form-mini.js (hunk 2 above)
Comment (ghost) [reply app/code/Magento/Search/view/frontend/web/js/form-mini.js]: @VladimirZaets I have applied the changes

**Review by VladimirZaets — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @VladimirZaets, thank you for the review.
[ENGCOM-5126](https://jira.corp.magento.com/browse/ENGCOM-5126) has been created to process this Pull Request

**Comment by soleksii** [comment]
:heavy_check_mark: QA Passed

**Before:**

before

**After:**

after

**Pull request merged**